modules/fedimint-ln-common/src/lib.rs

```diff
@@ -347,11 +347,12 @@ impl LightningGatewayAnnouncement {
 /// Information a gateway registers with a federation
 #[derive(Debug, Clone, Serialize, Deserialize, Encodable, Decodable, PartialEq, Eq, Hash)]
 pub struct LightningGateway {
-    /// Channel identifier assigned to the mint by the gateway.
+    /// Unique per-federation identifier assigned by the gateway.
     /// All clients in this federation should use this value as
     /// `short_channel_id` when creating invoices to be settled by this
     /// gateway.
-    pub mint_channel_id: u64,
+    #[serde(rename = "mint_channel_id", alias = "federation_index")]
```

Comment:
Looks like we're renaming the rust struct property but using the `serde(rename)` to prevent renaming the serialized field, making the `serde(alias)` unneeded. However, I think it makes more sense to change the serialized field by using this instead:

`#[serde(alias = "mint_channel_id")]`

Reply:
This does not work. I did this initially, and the database backwards compatibility tests in `fedimint-ln-tests` fail. The renaming is necessary for these to succeed.

Reply:
Gotcha. Then can we get rid of the `serde(alias)` and just have `#[serde(rename = "mint_channel_id")]`?

Reply:
Sure, that works. Done